app/models/payments/ChargeType.scala

```diff
@@ -374,7 +373,9 @@ object ChargeType extends LoggerUtil {
     VatOA1stLPPLPI,
     VatOA2ndLPPLPI,
     VatPA1stLPPLPI,
-    VatPA2ndLPPLPI
+    VatPA2ndLPPLPI,
```

Comment:
These charges also need to be in interestChargeTypes in order to generate the correct breakdown screen.